frontend/src/components/viewers/ROCCurve.tsx

```diff
@@ -141,16 +146,39 @@ class ROCCurve extends Viewer<ROCCurveProps, ROCCurveState> {
           />
 
           {/* Lines */}
-          {datasets.map((data, i) => (
+          {datasets.map(
+            (data, i) =>
+              highlightIndex !== i && (
+                <LineSeries
+                  key={i}
+                  color={
+                    this.props.colors
+                      ? this.props.colors[i]
+                      : lineColors[i] || lineColors[lineColors.length - 1]
+                  }
+                  strokeWidth={2}
+                  data={data}
+                  onNearestX={(d: any) => this._lineHovered(i, d)}
+                  curve='curveBasis'
+                />
+              ),
+          )}
+
+          {/* Highlighted line, if present */}
+          {highlightIndex !== -1 && (
```

Comment:
The highlighted line is rendered last so it is displayed on top, and it is rendered with a greater `strokeWidth` to ensure it stands out. This highlight feature is triggered when hovering over the associated color legend item.